data/korath/korath.txt

```diff
@@ -211,10 +214,35 @@ fleet "Korath Ember Waste Raid"
 	outfitters "Korath Basics" "Korath Exiles"
 	personality
 		disables plunders opportunistic harvests
-	variant
+	variant 14
 		"Korath Raider (Ember)"
 		"Korath Chaser" 2
+	variant 2
+		"Korath Raider (Ember Shunt)"
+		"Korath Chaser" 2
```

Comment:
The idea is that Shunt-Strikes are more used in EW, and heat lances are in other areas, due to the heat and nature of the wastes.

Reply:
Well, that was more of a content-gating consideration more than a gameplay one.  Shunt-Strikes are kept out of human space to preserve that wonder when they are first encountered in Korath or Remnant space, whereas heat weapons are nothing new for human space.

If we can make a Blaze-Pike Raider manage its own heat and energy effectively enough to operate in the wastes, I think that's a great suggestion.

Reply:
Also, since human hulls are basically tissue-paper (and so many human weapons have more of a focus on shield damage) it makes some sense that a raid would avoid using the weapon that costs them shield points when raiding the faction that does lots of shield damage and occasionally spawns large numbers of hostiles to combat a raid.